akvo/iati/exports/elements/budget.py

```diff
@@ -39,9 +47,11 @@ def budget(project):
                 value_element.attrib['currency'] = budget_item.currency
 
             if budget_item.other_extra:
-                value_element.attrib['{http://akvo.org/iati-activities}label'] = budget_item.other_extra
-            elif budget_item.label.label:
-                value_element.attrib['{http://akvo.org/iati-activities}label'] = budget_item.label.label
+                value_element.attrib['{http://akvo.org/iati-activities}label'] = budget_item.\
```

Comment:
To avoid the horrid \\:
```python
    akvo_ns_label = '{http://akvo.org/iati-activities}label'
    ...
    value_element.attrib[akvo_ns_label] = budget_item.other_extra
```